src/adapters/rubicon.js

```diff
@@ -149,15 +156,20 @@ function RubiconAdapter() {
 </html>`;
 
   function handleRpCB(responseText, bidRequest) {
-    let responseObj = JSON.parse(responseText); // can throw
+    let responseObj;
+    try {
+      responseObj = JSON.parse(responseText);
+    } catch(err) {
+      throw new RubiconSyntaxError("error parsing rubicon response", err);
```

Comment:
`RubiconSyntaxError` only handles the first argument.  So looks like we lose anything in `err`